lib/app/features/feed/views/pages/article_details_page/article_details_page.dart

```diff
@@ -64,78 +57,84 @@ class ArticleDetailsPage extends HookConsumerWidget {
                     ),
-                    ScreenSideOffset.small(child: const TextEditorPreview()),
-                    const ArticleDetailsDateTopics(),
-                    ScreenSideOffset.small(
-                      child: CounterItemsFooter(
-                        eventReference: eventReference,
-                        bottomPadding: 10.0.s,
-                      ),
-                    ),
-                    Container(color: context.theme.appColors.primaryBackground, height: 8.0.s),
-                    SizedBox(height: 20.0.s),
-                    ScreenSideOffset.small(
-                      child: UserBiography(pubkey: articleEntity.masterPubkey),
-                    ),
-                    SizedBox(height: 4.0.s),
-                    const ArticleDetailsTopics(),
-                    SizedBox(height: 4.0.s),
+                    if (articleEntity.data.content.isNotEmpty) SizedBox(height: 20.0.s),
                     ScreenSideOffset.small(
-                      child: ArticleDetailsSectionHeader(
-                        title: context.i18n.article_page_from_author('Alicia Twen'),
-                        count: 10,
-                        trailing: GestureDetector(
-                          onTap: () {},
-                          child: Assets.svg.iconButtonNext.icon(),
-                        ),
-                      ),
-                    ),
-                    Padding(
-                      padding: EdgeInsets.symmetric(vertical: 16.0.s),
-                      child: const ArticlesCarousel(
-                        articleIds: [
-                          'fdsfds',
-                          'fdsfdsfsdfsd',
-                          'fdsfdsfsd',
-                        ],
-                      ),
-                    ),
-                    ScreenSideOffset.small(
-                      child: ArticleDetailsSectionHeader(
-                        title: ArticleTopic
-                            .values[0].name, // TODO: replace with first topic from articleEntity
-                        count: 3,
-                        trailing: GestureDetector(
-                          onTap: () {},
-                          child: Assets.svg.iconButtonNext.icon(),
-                        ),
-                      ),
-                    ),
-                    Padding(
-                      padding: EdgeInsets.symmetric(vertical: 16.0.s),
-                      child: const ArticlesCarousel(
-                        articleIds: [
-                          'fdsfds',
-                          'fdsfdsfsdfsd',
-                          'fdsfdsfsd',
-                        ],
-                      ),
+                      child: TextEditorPreview(content: articleEntity.data.content),
                     ),
+
+                    // TODO: Uncomment all those blocks when API is ready
+
+                    // const ArticleDetailsDateTopics(),
+                    // ScreenSideOffset.small(
```

Comment:
Do we need this commented code, it doesn't look like ready even if uncommented it

Reply:
Then uncomment UI and mock data.